generator/src/main/java/com/scottlogic/deg/generator/profile/constraints/atomic/AfterConstantTimeConstraint.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.scottlogic.deg.generator.profile.constraints.atomic;
+
+import com.scottlogic.deg.common.profile.Field;
+import com.scottlogic.deg.common.profile.HelixTime;
+import com.scottlogic.deg.generator.fieldspecs.FieldSpec;
+import com.scottlogic.deg.generator.fieldspecs.FieldSpecFactory;
+import com.scottlogic.deg.generator.restrictions.linear.Limit;
+import com.scottlogic.deg.generator.restrictions.linear.LinearRestrictions;
+import com.scottlogic.deg.generator.restrictions.linear.LinearRestrictionsFactory;
+import com.scottlogic.deg.generator.utils.Defaults;
+
+import java.time.LocalTime;
+
+public class AfterConstantTimeConstraint implements AtomicConstraint {
+
+    public final Field field;
+    public final HelixTime referenceValue;
```

Comment:
These helix times should be replaced by LocalTime once the validation is moved out of HelixTime.